lib/hca/enrollment_system.rb

```diff
@@ -853,10 +853,10 @@ def get_va_format(content_type)
       end
     end
 
-    def add_attachment(file, is_dd214)
+    def add_attachment(file, id, is_dd214)
       {
         'va:document' => {
-          'va:name' => 'Attachment',
+          'va:name' => "Attachment#{id}",
```

Comment:
Small thing, but would `Attachment_1` be clearer here? The `Attachment1` syntax just looks weird to me but maybe that's just a me thing 🤷 .

Reply:
I'm not sure! My only line of thinking was that it wasn't ruby specific code, so I didn't make it snake case. I can change it though, if it'll make it clearer!